control/action.yaml

```diff
@@ -0,0 +1,44 @@
+openapi: 3.1.0
+
+info:
```

Comment:
remove this object from this file as it is not required and will be overwritten

Reply:
done